frontend/src/metabase/visualizations/components/settings/ChartSettingTableColumns/TableColumnPanel/TableColumnPanel.styled.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "@emotion/styled";
+
+import { TableColumnInfoIcon } from "metabase/components/MetadataInfo/ColumnInfoIcon";
+
+export const ColumnInfoIcon = styled(TableColumnInfoIcon)`
+  padding: 0;
```

Comment:
The [icon shrinks](https://github.com/metabase/metabase/assets/6830683/df41a850-b802-4c16-a582-70877fb85f54) when there's not enough room for it

```suggestion
  padding: 0;
  flex: 0 0 auto;
```